modules/ROOT/pages/use-environment-properties-on-tests-cookbook.adoc

```diff
@@ -0,0 +1,70 @@
+= Testing with Environment Properties
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Assume that you have the following flow that obtains data from an external API and you need to consume it from a test environment when executing your MUnit tests:
+
+[source,xml]
+----
+<http:request-config name="HTTP_Request_configuration">
+	<http:request-connection host="sampleapi.com/api/v1/" port="80"/>
+</http:request-config>
+<flow name="requestApiFlow">
+	<http:request method="GET" config-ref="HTTP_Request_configuration" path="/employee"/>
+</flow>
+----
+
+To consume data from a QA API endpoint when running your MUnit tests.
+
+. Extract the HTTP Configuration values into a properties file for each environment where you will execute the validations. Those properties files must go under `src/main/resources`:
++
+.prod.properties
+[source,xml,linenums]
+----
+http.host=dummy.restapiexample.com/api/v1/
+http.port=80
+----
++
+.qa.properties
+[source,xml,linenums]
+----
+http.host=dummy.qax.restapiexample.com/api/v1/
+http.port=8081
+----
++
+You can set a properties file for each one of your environments.
```

Comment:
Do we need to repeat the sentence before the example: "...for each environment where you will...."